src/App.tsx

```diff
@@ -1,109 +1,66 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const options: number[] = [3, 5, 10, 20];
 
 export const App: React.FC = () => {
+  const [itemsPerPage, setItemsPerPage] = useState<number>(options[1]);
+  const [currentPage, setCurrentPage] = useState<number>(1);
```

Comment:
if u use a simple type as the default value - the typescript itself itself understands what type should be in the state
```suggestion
  const [itemsPerPage, setItemsPerPage] = useState(options[1]);
  const [currentPage, setCurrentPage] = useState(1);
```